kafka-client/src/main/java/dev/responsive/kafka/api/config/ResponsiveConfig.java

```diff
@@ -68,6 +68,10 @@ public class ResponsiveConfig extends AbstractConfig {
   public static final String CLIENT_SECRET_CONFIG = "responsive.client.secret";
   private static final String CLIENT_SECRET_DOC = "The client secret for authenticated access";
 
+  public static final String STORAGE_BACKEND_TYPE_CONFIG = "responsive.storage.backend.type";
```

Comment:
enums are nice for `switch` statements in the code b/c IntelliJ can autofill them - otherwise I don't have a strong preference so I can change it. I guess if the config is well enough encapsulated I shouldn't be checking it in too many places anyway ;)

Reply:
we can refactor this later since it's totally backwards compatible (note that the config itself is actually a String, I'm just maintaining it as an enum)